### ISSUE TITLE
"This block has encountered an error and cannot be previewed." when adding paragraph.

### ISSUE THREAD

**Issue by PiotrWachulec** [WordPress/gutenberg#21119]
**Describe the bug**
When I was trying to add paragraph to page, the message from title was shown.

**To reproduce**
1. Open existing page or create the new one.
2. Try to add paragraph element.

**Expected behavior**
I was expected that the paragraph section will be added.

**Screenshots**
![Zrzut ekranu z 2020-03-24 19-57-35](https://user-images.githubusercontent.com/11841233/77466863-36b8d580-6e0b-11ea-84be-aad94d59442a.png)
And error code from the above screenshot:
```
TypeError: m.find is not a function
    at index.js?ver=7a8c786…c2e61df45e20373a:49
    at Array.reduce (<anonymous>)
    at index.js?ver=7a8c786…c2e61df45e20373a:49
    at Object.useMemo (react-dom.min.js?ver=16.9.0:205)
    at useMemo (react.min.js?ver=16.9.0:32)
    at He (index.js?ver=7a8c786…c2e61df45e20373a:49)
    at index.js?ver=c3c27a4…e4fb6a2a549ad5bf2:8
    at je (react-dom.min.js?ver=16.9.0:78)
    at ph (react-dom.min.js?ver=16.9.0:215)
    at lh (react-dom.min.js?ver=16.9.0:126)
```

**Desktop (please complete the following information):**
 - OS: Linux Ubuntu 18.04
 - Browser: Chrome
 - Version: 80.0.3987.149

**Smartphone (please complete the following information):**
 - Not tested.

**Additional context**
 - Wordpress: 5.3.2–pl_PL
 - Gutenberg: 7.7.1

**Comment by talldan** [comment]
@PiotrWachulec I haven't been able to reproduce this, and I feel like there would be lots of reports if the paragraph block stopped working, which there haven't been.

This may be related to a plugin you have active. It'd be great if you could try deactivating plugins, testing if you still have the issue. If you don't then you could try reactivating them one at a time to see if you can reproduce an error.

For now I'll close the issue, but it can always be reopened if it's narrowed down to a block editor bug.

**Comment by PiotrWachulec** [comment]
Hi @talldan ,

Thanks for your response. I turned off all plugins on my website except Gutenberg and the problem still occurs. 

Can I do something more to help you to investigate this issue?

Thanks in advance!

**Comment by PiotrWachulec** [comment]
With Gutenberg 7.8.0 and 7.8.1 error still occurs.

**Comment by PiotrWachulec** [comment]
I found! I'm using [Hestia](https://themeisle.com/themes/hestia/) theme, version 2.5.6. When I changed theme to default Wordpress Twenty Twenty, error disapeared. @talldan can you check with that theme?

**Comment by talldan** [comment]
@PiotrWachulec That theme seems to be behind a sign-up wall. I think the best thing would be to contact Theme Isle's support about the issue.

**Comment by PiotrWachulec** [comment]
![Zrzut ekranu z 2020-03-30 11-53-47](https://user-images.githubusercontent.com/11841233/77899686-4017c700-727d-11ea-89fa-446747686736.png)
When you will click **Download button** from page from my previous comment and you will see form from screenshot above, you can click _**Thanks, I want to just download the zip**_ (under **Subscribe and download** button). You can install theme directly from Wordpress from Theme search. Sorry for the polish language on screenshots ;)
![Zrzut ekranu z 2020-03-30 12-00-08](https://user-images.githubusercontent.com/11841233/77900225-04c9c800-727e-11ea-907a-6e50ec848b8d.png)

**Comment by talldan** [comment]
@PiotrWachulec Ok, I installed the theme from the theme directory. I had no issues adding paragraphs in the editor.

**Comment by PiotrWachulec** [comment]
Hi @talldan ,

Thank you for your response. Unfortunately I reproduced the issue on the fresh installation of Wordpress on the other hosting. I installed Wordpress 5.3.2-pl_PL and then I installed Gutenberg and Hestia theme and that's all. I got situation described in issue. Can I help you in some way in reproducing the issue? Some logs or something like that?

Thanks in advance.

**Comment by talldan** [comment]
@PiotrWachulec @maate33 - given that this seems to only be happening with the Hestia theme, I'd recommend reaching out to the theme developer about the issue.

**Comment by PiotrWachulec** [comment]
Hi @talldan ,

Thank you for your help. You are right, fix for this should be today by Hestia team: [https://wordpress.org/support/topic/problems-with-blocks-in-hestia-after-wordpress-updating-to-wordpress-4-5_nl/](https://wordpress.org/support/topic/problems-with-blocks-in-hestia-after-wordpress-updating-to-wordpress-4-5_nl/)

CC: @maate33

**Comment by PiotrWachulec** [comment]
Issue solved with update Hestia theme to version 2.5.7.

**Comment by talldan** [comment]
Glad to hear it was resolved 👍
